test/functionalTest/cases/4286_update_format_for_notification_messages/update_format_for_notification_messages_custom_ngsi_patching.test

```diff
@@ -0,0 +1,211 @@
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+update format for notification messages custom ngsi patching
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# 01. Create NGSIv2 subscription for E1/A1 with attrsFormat set to simplifiedNormalized with httpCustom and ngsi field adding a new attribute
+# 02. Create NGSIv2 subscription for E1/A1 with attrsFormat set to simplifiedkeyValues with httpCustom and ngsi field adding a new attribute
+# 03. Create entity E1/T1
+# 04. Dump accumulator to see notifications
+#
+
+
+echo "01. Create NGSIv2 subscription for E1/A1 with attrsFormat set to simplifiedNormalized with httpCustom and ngsi field adding a new attribute"
+echo "==========================================================================================================================================="
+payload='
+{
+  "subject": {
+    "entities": [
+      {
+        "id": "E1"
+      }
+    ],
+    "condition": {
+      "attrs": [ "A1" ]
+    }
+  },
+  "notification": {
+    "httpCustom": {
+      "url": "http://localhost:'$LISTENER_PORT'/notify",
+      "ngsi": {
+        "id": "E1",
+        "type": "T1",
+        "A2": {
+          "value": 1,
+          "type": "Number"
+        }
+      }
+    },
+    "attrs": [ "A1" ],
+    "attrsFormat": "simplifiedNormalized"
+  },
+  "expires": "2050-04-05T14:00:00.00Z"
+}'
+orionCurl --url /v2/subscriptions --payload "$payload"
+echo
+echo
+
+
+echo "02. Create NGSIv2 subscription for E1/A1 with attrsFormat set to simplifiedKeyValues with httpCustom and ngsi field adding a new attribute"
+echo "=========================================================================================================================================="
+payload='
+{
+  "subject": {
+    "entities": [
+      {
+        "id": "E1"
+      }
+    ],
+    "condition": {
+      "attrs": [ "A1" ]
+    }
+  },
+  "notification": {
+    "httpCustom": {
+      "url": "http://localhost:'$LISTENER_PORT'/notify",
+      "ngsi": {
+        "id": "E1",
+        "type": "T1",
```

Comment:
```suggestion
```

You only need the A3 part. You can check documentation here: https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#ngsi-payload-patching

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4298/commits/7bf26136d4ee0b7fa8abd10773fcf7ef1e3e043a